src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+/* CONTAINER */
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+/* LOGO */
+
+.home__link {
+  display: flex;
+  align-items: center;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+}
+
+/* LINKS */
+
+.links {
+  display: flex;
+  list-style: none;
```

Comment:
This is a good example of how to style elements using `.class` instead of tag names. Keep it up!